ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml

```diff
@@ -85,13 +85,7 @@ tests:
       BASE_DOMAIN: qe.devcluster.openshift.com
       COMPUTE_NODE_REPLICAS: "9"
       COMPUTE_NODE_TYPE: m5.xlarge
-      CONFIG: config/standard-classic-rosa.yml
```

Comment:
@jtaleric Do you think we should rerun this test to confirm results as we are deleting these params here?

Reply:
These parameters are for managed -- this is for self-managed (not ROSA)